lib/content-services/src/lib/content-metadata/services/property-groups-translator.service.spec.ts

```diff
@@ -428,5 +429,27 @@ describe('PropertyGroupTranslatorService', () => {
             expect(cardViewProperty.key).toBe('properties.fk:emperor');
             expect(cardViewProperty.editable).toBe(false);
         });
+
+        it('should translate properly the value attribute for d:long', () => {
+            property.dataType = 'd:long';
+
+            propertyValues = { 'FAS:PLAGUE': '1024' };
+            const cardViewGroup = service.translateToCardViewGroups(propertyGroups, propertyValues, null);
+
+            const cardViewProperty: CardViewLongItemModel = cardViewGroup[0].properties[0] as CardViewLongItemModel;
+            expect(cardViewProperty instanceof CardViewLongItemModel).toBeTruthy();
```

Comment:
I'm not so sure if this assertion is really needed. The cardViewProperty variable is explicitly defined as a type of CardViewLongItemModel, and then the value being assigned to it during initialising is also type casted as CardViewLongItemModel. So having an assertion to verify its type is not validating the code in the component, rather the code in the test case

Reply:
@swapnil-verma-gl You're right, checking instanceof is pointless with type defined. My bad, it was copy-pasted. Fixed similar occurrences in the whole file.